### PR TITLE
<Table/> - Remove box shadow from row if expanded row details

### PR DESCRIPTION
**Pull request by alexandraseri**
### 🔦 Summary

When DataTable receives rowDetails, upon selection of a row with rowDetails, there should not be box-shadow between the row and the expanded details row.

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [X] 👨‍🎨 UX change is approved by the Design System UX Ayelet Reuveni
- 📚 Change is documented
  - [X] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5742/

File: src/Table/DataTable/DataTable.js [hunk 1]
```diff
@@ -374,11 +374,17 @@ class DataTable extends React.Component {
         ? getStickyColumnStyle(columns, column)
         : undefined;
 
+    const rowDetailsExpandedStyle =
+      !!this.state.selectedRows.get(rowData) && rowDetails(rowData)
+        ? { boxShadow: 'none' }
+        : {};
+
```
Comment (EgoziE) [inline src/Table/DataTable/DataTable.js:381]: Can it be just part of `const classes` above?
with a dedicated css class

File: src/Table/docs/examples/TableRowDetailsExample.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
```
Comment (EgoziE) [inline src/Table/docs/examples/TableRowDetailsExample.js:1]: It's a bit difficult to understand the purpose of this example without reading the code.
I have a suggestion to fix it:

```
class TableRowDetailsExample extends React.Component {
  state = {
    data: [
      { firstName: 'Meghan', lastName: 'Bishop', expandable: true },
      { firstName: 'Sara', lastName: 'Porter', expandable: false },
      { firstName: 'Deborah', lastName: 'Rhodes', expandable: true },
      { firstName: 'Walter', lastName: 'Jenning', expandable: true },
    ],
  };

  renderRowDetails(row) {
    if (row.expandable)
      return (
        <Box padding="20px">
          <Text>
            {`This is an expandible space where you see the row details for ${row.firstName} ${row.lastName}.`}
          </Text>
        </Box>
      );
  }

  render() {
    return (
      <Table
        data={this.state.data}
        columns={[
          { title: 'First', render: row => row.firstName },
          { title: 'Last', render: row => row.lastName },
          {
            title: 'expandable',
            render: row =>
              row.expandable ? (
                <Badge size="small">Expandable</Badge>
              ) : (
                <Badge size="small" skin="danger">
                  Not Expandable
                </Badge>
              ),
          },
        ]}
        rowDetails={row => this.renderRowDetails(row)}
      >
        <Table.Content />
      </Table>
    );
  }
}
```

File: src/Table/docs/index.story.js [hunk 3]
```diff
@@ -170,6 +171,13 @@ export default {
               source: TableStickyScrollExampleRaw,
               compact: true,
             },
+            {
+              title: 'Table with Expandable Rows',
+              description:
+                'A basic table with data, having row details for some rows.',
```
Comment (EgoziE) [inline src/Table/docs/index.story.js:177]:
```suggestion
                'A table with expandable rows. A row is expandable using the `renderRowDetails` prop'.,
```

**Pull request closed without merging**